src/css/styles.css

```diff
@@ -0,0 +1,77 @@
+/* *** style guide *** */
+/* font-family: 'Alice', serif; */
+/* font-family: 'Raleway', sans-serif; */
+/*colors:  navy blue (#026B81), turquoise (#03B0AA), pink (#D49C96), dusty rose (#CBBDBE)*/
+
+
+/* *** General *** */
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    font-family: 'Raleway', sans-serif;
+}
+
+/* *** Grid *** */
+
+/* *** Header *** */
+.header__left {
+
+}
+
+.header__right--text {
```

Comment:
Would love to hear more about the reasoning behind this naming scheme

Reply:
Right! This is BEM -- let's talk about it in our meeting today.